bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -899,6 +899,7 @@ private String quantityTypeToDPTValue(QuantityType<?> qt, int mainNumber, int su
                         case 8:
                             return translator3BitControlled.getControlBit() ? UpDownType.DOWN : UpDownType.UP;
                     }
+                    break;
```

Comment:
I see that a break is missing for cases 1 and 2. Please confirm that it is expected.

Reply:
case 1 has default case with return statement, case 2 also ends with a return statement -> LGTM